app/controllers/api/v1/group_order_articles_controller.rb

```diff
@@ -0,0 +1,95 @@
+class Api::V1::GroupOrderArticlesController < Api::V1::BaseController
+  include Concerns::CollectionScope
+
+  before_action :require_ordergroup
+  before_action :require_minimum_balance, only: [:create, :update] # destroy is ok
+  before_action :require_enough_apples, only: [:create, :update] # destroy is ok
+  # @todo allow decreasing amounts when minimum balance isn't met
+
+  def index
+    render_collection search_scope
+  end
+
+  def show
+    goa = scope.find(params.require(:id))
+    render_goa_with_oa(goa)
+  end
+
+  def create
+    GroupOrderArticle.transaction do
+      oa = order_articles_scope.find(create_params.require(:order_article_id))
+      go = current_ordergroup.group_orders.find_or_create_by!(order_id: oa.order_id)
+      goa = go.group_order_articles.create!(order_article: oa)
+      goa.update_quantities((create_params[:quantity] || 0).to_i, (create_params[:tolerance] || 0).to_i)
```

Comment:
does `create_params` do not provide any method for getting with an default value?

Reply:
Not that I could find (`fetch` only does it for a single key). Strong parameters is merely about passing and cleaning parameters, not about giving defaults. I think it's fine to explicitely put it here (there could be different behaviour if one is `nil` in other places).

Reply:
if you say so, i'm ok with it